src/qt/sendcoinsdialog.cpp

```diff
@@ -321,7 +328,7 @@ void SendCoinsDialog::on_sendButton_clicked()
 
     // add total amount in all subdivision units
     questionString.append("<hr />");
-    CAmount totalAmount = nTotalAmount + txFee;
+    CAmount totalAmount = nTotalAmount + (rcp.fSubtractFeeFromAmount ? -txFee : txFee);
```

Comment:
should this be ?:
```cpp
CAmount totalAmount = nTotalAmount + (rcp.fSubtractFeeFromAmount ? 0 : txFee);
```

Reply:
without subtracting fee:

<img width="419" alt="image" src="https://user-images.githubusercontent.com/24814046/106395447-99eb9f00-6402-11eb-8c5f-b5adb64eeae0.png">

with subtracting fee:

<img width="426" alt="image" src="https://user-images.githubusercontent.com/24814046/106395457-acfe6f00-6402-11eb-990d-386d0a9bfb15.png">

Reply:
The total amount should be equal to 1 NAV right? Cause the fee is subtracted from the amount, but it's still needed.

Reply:
fixed in last commit:

without subtracting:

<img width="416" alt="image" src="https://user-images.githubusercontent.com/24814046/106395563-2a29e400-6403-11eb-870f-81351a7d3024.png">

with subtracting:

<img width="423" alt="image" src="https://user-images.githubusercontent.com/24814046/106395568-3746d300-6403-11eb-8b70-f2b5b9b76720.png">

Reply:
> fixed in last commit:
> 
> without subtracting:
> 
> <img alt="image" width="416" src="https://user-images.githubusercontent.com/24814046/106395563-2a29e400-6403-11eb-870f-81351a7d3024.png">
> 
> with subtracting:
> 
> <img alt="image" width="423" src="https://user-images.githubusercontent.com/24814046/106395568-3746d300-6403-11eb-8b70-f2b5b9b76720.png">

Nice, will test in a few minutes